velox/connectors/hive/HiveConnector.h

```diff
@@ -327,4 +327,46 @@ class HiveHadoop2ConnectorFactory : public HiveConnectorFactory {
       : HiveConnectorFactory(kHiveHadoop2ConnectorName) {}
 };
 
+class HivePartitionFunctionSpec : public core::PartitionFunctionSpec {
+ public:
+  HivePartitionFunctionSpec(
+      int numBuckets,
```

Comment:
I wonder if we need numBuckets parameter. Isn't it always equal to bucketToPartition.size()?